app/src/main/java/com/willowtree/vocable/settings/customcategories/CustomCategoryPhraseListFragment.kt

```diff
@@ -3,33 +3,45 @@ package com.willowtree.vocable.settings.customcategories
 import android.os.Bundle
 import android.view.View
 import androidx.core.os.bundleOf
+import androidx.core.view.isVisible
+import androidx.lifecycle.ViewModelProviders
 import androidx.recyclerview.widget.GridLayoutManager
 import com.willowtree.vocable.BaseFragment
+import com.willowtree.vocable.BaseViewModelFactory
 import com.willowtree.vocable.BindingInflater
 import com.willowtree.vocable.R
 import com.willowtree.vocable.databinding.FragmentCustomCategoryPhraseListBinding
+import com.willowtree.vocable.room.Category
 import com.willowtree.vocable.room.Phrase
+import com.willowtree.vocable.settings.EditCategoriesViewModel
 import com.willowtree.vocable.settings.customcategories.adapter.CustomCategoryPhraseAdapter
 import com.willowtree.vocable.utils.ItemOffsetDecoration
 
 class CustomCategoryPhraseListFragment : BaseFragment<FragmentCustomCategoryPhraseListBinding>() {
 
     companion object {
         private const val KEY_PHRASES = "KEY_PHRASES"
+        private const val KEY_CATEGORY = "KEY_CATEGORY"
 
-        fun newInstance(phrases: List<Phrase>): CustomCategoryPhraseListFragment {
+        fun newInstance(
+            phrases: List<Phrase>,
+            category: Category
+        ): CustomCategoryPhraseListFragment {
             return CustomCategoryPhraseListFragment().apply {
-                arguments = bundleOf(KEY_PHRASES to ArrayList(phrases))
+                arguments = bundleOf(KEY_PHRASES to ArrayList(phrases), KEY_CATEGORY to category)
```

Comment:
Do we still need this, now that we're using Navigation?  New fragment arguments should be added to that.

Reply:
So this is a fragment that we're adding to a `FragmentStateAdapter`, so we're not technically navigating to it, we're just using it to populate an adapter. I don't know if we can use the nav component for that.